Client/Frontend/Browser/SimpleToast.swift

```diff
@@ -19,8 +19,22 @@ struct SimpleToastUX {
     static let BottomToolbarHeight = CGFloat(45)
 }
 
-struct SimpleToast {
-    func showAlertWithText(_ text: String, image: String, bottomContainer: UIView) {
+class SimpleToast {
```

Comment:
If possible, unless inherited somewhere 🙏 
```suggestion
final class SimpleToast {
```